### ISSUE TITLE
SKR E3 Mini v1.2 and Octoprint

### ISSUE THREAD

**Issue by davehardy20** [bigtreetech/BIGTREETECH-SKR-mini-E3#41]
Whenever I plugin the  USB cable to the board from my OctoPi there seems to be connection issues, sometimes its fine , but mostly  1 of 2 things happens.
1. the E3 Mini board crashes
2. the board seems fine and can print from OctoPi, however the display just goes haywire.
This OctoPi  is / was perfectly fine with the Creality 1.1.5 board I used before the E3 Mini v1.2
![IMG_20191013_153623](https://user-images.githubusercontent.com/874172/66718972-c466b280-ede1-11e9-94fc-be69edc48b92.jpg)

I've had a look on the OctoPrint forums but there doesn't seem to be anyone else reporting this issue.
Is this a known problem or a new issue?

**Comment by lococnc** [comment]
Mine works fine. I do have the 5v taped off so it is not powering the SKR board.

**Comment by davehardy20** [comment]
@lococnc your suggestion seems to have done the trick, I taped off the 5v connector in the USB A plug and things look good now. Thanks
I would guess that the newer 32bit chip and newer electronics are more susceptible to the 5v feed from the PI, whereas the Creality 8bit electronics may not be so sensitive. Just a guess though.

**Comment by ManiRX100** [comment]
@davehardy20 I am also having the issue with Cura but the USB works with Pronterface and Simplify 3D. How do you tap off the 5V connector? Remove the 5V supply line from the USB ?

**Comment by pusewicz** [comment]
@ManiRX100 Simply put a bit of insulation tape on the pin in the USB plug.

![](https://k61.kn3.net/0/F/D/A/7/1/09A.jpg)

**Comment by davehardy20** [comment]
Just as @pusewicz  says, but I used electrical insulation tape. Works just fine

**Comment by Thinkersbluff** [comment]
@pusewicz: Thanks for the tip on how to tape-off the 5VDC pin on a USB connector.
I just had to order a 2nd SKR mini e3 v1.2, after apparently having somehow fried the X-axis driver within 3 weeks of the upgrade.  
This board exhibits various strange behaviours if powered by the Pi or PC through the USB cable before turning on the printer's main power.   I even found all my motors super hot, when I came back two days after leaving the Pi on and the printer power off.  Apparently, the TMC Connection Error did not trigger in that case, either.
Hopefully, this work around protects my new board.
